hazelcast/src/test/java/com/hazelcast/config/alto/AltoSocketConfigTest.java

```diff
@@ -77,34 +67,16 @@ public void testClientPortsNotEnough() {
     public void testClientPortsWith3Members() {
         getAltoSocketConfig().setPortRange("13000-14000");
         HazelcastInstance[] hz = createHazelcastInstances(config, 3);
-        ArrayList<Integer> expectedPorts = new ArrayList<>();
-
-        // ports of hz0
-        for (int i = 0; i < EVENTLOOP_COUNT; i++) {
-            expectedPorts.add(13000 + i);
-        }
-        assertEquals(expectedPorts, getClientPorts(hz[0]));
-
-        // ports of hz1
-        for (int i = 0; i < EVENTLOOP_COUNT; i++) {
-            expectedPorts.set(i, expectedPorts.get(i) + EVENTLOOP_COUNT);
-        }
-        assertEquals(expectedPorts, getClientPorts(hz[1]));
-
-        // ports of hz2
-        for (int i = 0; i < EVENTLOOP_COUNT; i++) {
-            expectedPorts.set(i, expectedPorts.get(i) + EVENTLOOP_COUNT);
-        }
-        assertEquals(expectedPorts, getClientPorts(hz[2]));
+        assertClientPorts(EVENTLOOP_COUNT, 13000, 14000, hz);
     }
 
     @Test
-    public void testConfigBounds() {
+    public void testConfigValidation() {
         AltoSocketConfig altoSocketConfig = getAltoSocketConfig();
-        assertThrows(InvalidConfigurationException.class, () -> altoSocketConfig.setReceiveBufferSize(1 << 15 - 1));
-        assertThrows(InvalidConfigurationException.class, () -> altoSocketConfig.setReceiveBufferSize(1 << 30 + 1));
-        assertThrows(InvalidConfigurationException.class, () -> altoSocketConfig.setSendBufferSize(1 << 15 - 1));
-        assertThrows(InvalidConfigurationException.class, () -> altoSocketConfig.setSendBufferSize(1 << 30 + 1));
+        assertThrows(InvalidConfigurationException.class, () -> altoSocketConfig.setReceiveBufferSizeKb(1 << 5 - 1));
```

Comment:
Please don't use this bit-shifting logic to express a number. If you want to use 32 bytes, then just say 32 and not 1<<5.

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/16aaf911c270da78592443f9d81c145858b9459e